pages/glossary/programatically-determined-link-context.md

```diff
@@ -0,0 +1,19 @@
+--
+title: Programmatically Determined Link Context
+key: programmatically-determined-link-context
+--
+
+In HTML, information that is programmatically determinable from a link includes [text nodes][] that are [included in the accessibility tree][] with a non-empty ("") [accessible name][] and are
+
+- in the same paragraph (HTML `p` element) as the link;
+- in the same list (HTML element with the semantic role of `listitem`) as the link;
```

Comment:
This is confusing. Why not say in the same listitem, if that's the role you're after.